lib/contracts/builtin_contracts.rb

```diff
@@ -393,6 +393,25 @@ def validate_hash(hash)
       end
     end
 
+    # Use this to specify the Hash characteristics with strict check of keys'
```

Comment:
Could you elaborate on "strict check of keys' presence"? It would be nice to say "the contracts fails if there are any extra keys that don't have contracts on them".